cocos/core/data/deserialize.ts

```diff
@@ -993,7 +1009,7 @@ export function deserialize (data: IFileData | string | CCON | any, details: Det
     details = details || Details.pool.get();
     let res;
 
-    if (!BUILD && !(PREVIEW && isCompiledJson(data))) {
+    if (!FORCE_COMPILED && !isCompiledJson(data)) {
```

Comment:
Why to change `BUILD` macro to `FORCE_COMPILED` which is always `false`?
Currently , `deserialize-dynamic.ts` will not be cut while `building` project.